src/components/CourtDateFilter/CourtDateFilterOptions.tsx

```diff
@@ -20,7 +23,16 @@ const formatNamedDateRange = (namedDateRange: string): string => {
 const labelForDateRange = (namedDateRange: string): string =>
   ["Today", "Yesterday"].includes(namedDateRange) ? namedDateRange : formatNamedDateRange(namedDateRange)
 
-const CourtDateFilterOptions: React.FC<Props> = ({ dateRange, dispatch }: Props) => {
+// TODO- update radio button logic for nested radios. Move away from GOV UK condition classes and use "states" instead.
```

Comment:
We still have a TODO here.

Reply:
This TODO will be worked during the BICAWS-2616 refactoring to use states rather than GovUK conditional classes.